docs/content/ChoiceGroup.mdx

```diff
@@ -0,0 +1,382 @@
+  const [selectedRadioValue, setSelectedRadioValue] = React.useState('')
+
+  const handleCheckboxChange = e => {
+    const {value, checked} = e.currentTarget
+
+    if (checked) {
+      setSelectedCheckboxValues([...selectedCheckboxValues, value])
+      return
+    }
+
+    setSelectedCheckboxValues(selectedCheckboxValues.filter(selectedValue => selectedValue !== value))
+  }
+
+  const handleRadioChange = e => {
+    const {value, checked} = e.currentTarget
+
+    if (checked) {
+      setSelectedRadioValue(value)
+    }
+  }
+
+  return (
+    <Box display="grid" sx={{gap: 3}}>
+      <Box display="grid" sx={{gap: 1}}>
+        <ChoiceGroup>
+          <ChoiceGroup.Label>Checkboxes</ChoiceGroup.Label>
+          <FormControl>
+            <Checkbox value="checkOne" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox one</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Checkbox value="checkTwo" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox two</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Checkbox value="checkThree" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox three</FormControl.Label>
+          </FormControl>
+        </ChoiceGroup>
+
+        {Boolean(selectedCheckboxValues.length) && (
+          <div>The selected checkbox values are {selectedCheckboxValues.join(', ')}</div>
+        )}
+      </Box>
+
+      <Box display="grid" sx={{gap: 1}}>
+        <ChoiceGroup>
+          <ChoiceGroup.Label>Radios</ChoiceGroup.Label>
+          <FormControl>
+            <Radio name="radioChoices" value="radioOne" onChange={handleRadioChange} />
+            <FormControl.Label>Radio one</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Radio name="radioChoices" value="radioTwo" onChange={handleRadioChange} />
+            <FormControl.Label>Radio two</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Radio name="radioChoices" value="radioThree" onChange={handleRadioChange} />
+            <FormControl.Label>Radio three</FormControl.Label>
+          </FormControl>
+        </ChoiceGroup>
+
+        {selectedRadioValue && <div>The selected radio value is {selectedRadioValue}</div>}
+      </Box>
+    </Box>
+  )
+}
+
+render(<WithOnChangeHandlers />)
+```
+
+### Checkbox group
+
+```jsx live
+<ChoiceGroup>
+  <ChoiceGroup.Label>Choices</ChoiceGroup.Label>
+  <FormControl>
+    <Checkbox />
+    <FormControl.Label>Choice one</FormControl.Label>
+  </FormControl>
+  <FormControl>
+    <Checkbox />
+    <FormControl.Label>Choice two</FormControl.Label>
+  </FormControl>
+  <FormControl>
+    <Checkbox />
+    <FormControl.Label>Choice three</FormControl.Label>
+  </FormControl>
+</ChoiceGroup>
+```
+
+### Disabled
+
+```jsx live
+<ChoiceGroup disabled>
```

Comment:
FYI some cursor inconsistencies here

https://user-images.githubusercontent.com/13340707/154352388-30501123-db97-4abf-a42a-9b06ee9f3ec5.mov

Reply:
Good catch. I'll fix that.